servers/src/main/java/tachyon/master/TachyonMaster.java

```diff
@@ -181,6 +181,10 @@ protected TachyonMaster() {
       MasterContext.getMasterSource().registerGauges(this);
       mMasterMetricsSystem = new MetricsSystem("master", MasterContext.getConf());
       mMasterMetricsSystem.registerSource(MasterContext.getMasterSource());
+
+      mWebServer =
```

Comment:
This needed to move here so that trying to connect to `mWebServer.getBindHost()` after the constructor wouldn't throw NPE.